.azuredevops/modulePipelines/ms.analysisservices.servers.yml

```diff
@@ -6,7 +6,11 @@ parameters:
     type: boolean
     default: true
   - name: prerelease
-    displayName: Publish prerelease module
+    displayName: Publish prerelease
+    type: boolean
+    default: false
+  - name: publishAll
+    displayName: Publish module and all child modules
```

Comment:
```suggestion
    displayName: Force publish module and all child modules
```

Let's have a word on that in the group. For now just a small adjustment proposal ;)

Reply:
This makes more sense, though I removed the "Force" wording based on initial feedback. In retrospect, that might have been based on the usage of `force` in powershell.

Reply:
Agree. I was trying to find a different wording, but couldn't find one. :D

Reply:
We opted to remove this feature, so closing this topic.